docs/development/extensions-contrib/k8s-jobs.md

```diff
@@ -286,4 +286,4 @@ To do this, set the following property.
 |--------|-----------------|-----------|-------|--------|
 |`druid.indexer.runner.k8sAndWorker.workerTaskRunnerType`|`String`|Determines whether the `httpRemote` or the `remote` task runner should be used in addition to the Kubernetes task runner.|`httpRemote`|No|
 |`druid.indexer.runner.k8sAndWorker.sendAllTasksToWorkerTaskRunner`|`boolean`| Whether to send all the tasks to the worker task runner. If this is set to false all tasks will be sent to Kubernetes|`false`|No|
-
+|`druid.indexer.runner.k8sAndWorker.taskTypeToWorkerTaskRunnerOverrides`|`Map<String, Boolean>`|Json key value pairs of task types and a boolean for whether or not the task should be sent to the worker task runner. If the task type is not in this list, tasks will be assigned to a worker based on the `sendAllTasksToWorkerTaskRunner` config. Example: `{'index_kafka': true}`|`{}`|No|
```

Comment:
i think i would prefer exposing this as a new class (K8sRunnerSelectStrategy) or something like this. I can imagine many other ways someone might want to choose where to send tasks (like WorkerStrategy). Otherwise I think the # of configs could spiral and it would be confusing which one takes priority.

So to support splitting by task type introduce a new TaskTypeK8sRunnerSelectStrategy and a config
druid.indexer.runner.k8sAndWorker.strategy="taskType" that defaults to strategy="all" (AllK8sRunnerSelectStrategy) which can send tasks to either one or the other.